lxd/instance.go

```diff
@@ -27,6 +24,7 @@ import (
 	"github.com/lxc/lxd/shared"
 	log "github.com/lxc/lxd/shared/log15"
 	"github.com/lxc/lxd/shared/logger"
+	"github.com/pkg/errors"
```

Comment:
We keep external imports in their own block of imports between built-in packages and our own packages.